### ISSUE TITLE
Known issues installing on Ubuntu 20.04 (focal fossa)

### ISSUE THREAD

**Issue by ykoji8681** [s-tip/stip-common#24]
### 1. Mysql 5.7 focal package does not exist.
* Workaround 1: Update mysql version in S-TIP to 8.0
However, following error occurred with mysql 8.0 in setup_rs.sh.
```
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that
 corresponds to your MySQL server version for the right syntax to use near 'identified by 'stip' 
with grant option' at line 1
```
> grant all privileges on s_tip.* to stip@"%" identified by 'stip' with grant option;

It seems that automated user creation is not allowed in 8.0. Do we need to add user creation command before grant?

https://www7390uo.sakura.ne.jp/wordpress/archives/456

* Workaround 2: wait until mysql 5.7 will appear in focal apt repository

* Workaround 3: Install 5.7 with deb package

### 2. Mongo focal package does not exist.
* Workaround 1: use bionic package
Modify setup_rs.sh as follows:
```
 if [ -f /etc/lsb-release ]; then
     . /etc/lsb-release
     echo "Ubuntu Codename: $DISTRIB_CODENAME"
+    if [ $DISTRIB_CODENAME = "focal" ]; then
+        echo "Distribution focal detected. Use BIONIC mongo package instead of focal temporarily."
+        DISTRIB_CODENAME=bionic
+    fi
```
* Workaround 2: wait until focal package will appear in mongo repository
http://repo.mongodb.org/apt/ubuntu/dists/

**Comment by ykoji8681** [comment]
* Problem 1 (Mysql) was resolved in S-TIP v1.0 (today's release).
* Problem 2 (Mongo) still exists. Modify `setup_rs.sh` script as above before installing on your environment.

**Comment by ykoji8681** [comment]
Merged in #30.
